doc/api_methods/AddSubscriber.md

```diff
@@ -0,0 +1,44 @@
+[back to list](../Readme.md)
+
+# Add Subscriber
+
+## `array addSubscriber(array $subsriber [, array $list_ids = [], array $options = []])`
+
+This method allows to create a subscriber, add him into lists and handles confirmation email and admin notification email sending and welcome email scheduling.
+
+If sign-up confirmation is enabled in the MailPoet settings a subscriber is created with status `unconfirmed` otherwise the status is set to `subscribed`.
+
+*A confirmation email* is an email which is sent to a subscriber so that he can confirm his subscription. It is sent only if sign-up confirmation is enabled in the MailPoet settings.
+*A welcome email* is an automatic email which is sent to a new subscriber. This email is scheduled only if sign-up confirmation is disabled and some welcome email is configured for some of given lists.
```

Comment:
You could maybe mention that this email will be sent after the user confirms the subscription if double-opt-in is active.